jquery.flot.touchNavigate.js

```diff
@@ -34,44 +34,44 @@
             if (!twoTouches) {
                 prevTapX = prevPanX;
                 prevTapY = prevPanY;
-                prevPanX = e.touches[0].clientX;
-                prevPanY = e.touches[0].clientY;
+                prevPanX = e.touches[0].pageX;
+                prevPanY = e.touches[0].pageY;
             } else {
                 prevDist = pinchDistance(e);
-                prevPanX = (e.touches[0].clientX + e.touches[1].clientX) / 2;
-                prevPanY = (e.touches[0].clientY + e.touches[1].clientY) / 2;
+                prevPanX = (e.touches[0].pageX + e.touches[1].pageX) / 2;
+                prevPanY = (e.touches[0].pageY + e.touches[1].pageY) / 2;
```

Comment:
There should be at least a test proving .pageXY it the right choice. Maybe copy-paste one of the existing tests and create a large div above the plot to force the drawing away from the top of the screen.